### PR TITLE
Add content - humanhealth featured content

### PR DESCRIPTION
**Pull request by nickumia-reisys**
Related to
- https://github.com/GSA/data.gov/issues/3891

Replaces
- https://github.com/GSA/datagov-website/blob/main/www.data.gov/climate/humanhealth/humanhealth-featured-content/index.html

### CODE REVIEW

File: posts/2014-02-26-humanhealth-featured-content.md
Comment (nickumia-reisys) [inline posts/2014-02-26-humanhealth-featured-content.md:25]:
```suggestion
The [Nutrient and Coliform Loading (NCL)](https://products.coastalscience.noaa.gov/ncl/) is a compilation of available fecal coliform bacteria, fecal streptococci bacteria, and nutrient loading data. Loading for contaminants other than fecal coliform bacteria and nutrients are included when values occurred coincidentally within the same literature. This website can be used to quickly identify available measures of loading from different land use (bacteria and nutrients) and animals (bacteria only). Increasing coastal development causes higher levels of both contaminant types to enter estuaries and affects water quality including both harvestability of shellfish as well as recreational uses.
```

**Comment by nickumia-reisys** [comment]
Don't merge this until we figure out if the `back to climate homepage` link is useful.

**Comment by nickumia-reisys** [comment]
@Jin-Sun-tts @btylerburton This is ready to go 🪄

**Review by Jin-Sun-tts — APPROVED** [review]

**Pull request merged**
